js/block.js

```diff
@@ -6699,6 +6750,168 @@ function Block(protoblock, blocks, overrideName) {
             };
         }
     };
+    this._piemenuHorizontalNumber = function(wheelValues, selectedValue) {
```

Comment:
Rather than replicate the number pie menu with just the change in orientation, why not just check the block name inside _piemenuNumber to set the orientation??? would be much less code and easier to maintain.

Reply:
I wasn't too sure since it would change all of the pie menus orientation. But will do and thanks for the suggestions!

Reply:
IThanks for the suggestions; I will do.